.babelrc

```diff
@@ -4,27 +4,4 @@
     "flow"
   ],
   "retainLines": true,
-  "plugins": [
-    [
-      "module-resolver",
-      {
-        "root": [
```

Comment:
Agreed - however if it's bothering you @patw0929 because of the linter I'm happy to compromise and remove it.